tox.ini

```diff
@@ -9,17 +9,18 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
 
 [testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
+commands = nosetests --with-coverage --cover-package=googleapiclient --nocapture --cover-erase --cover-tests --cover-branches --cover-min-percentage=85
 
 [testenv:py27]
-commands = nosetests
+commands = {[testenv:py26]commands}
```

Comment:
What's the significance of "testenv:py26" on lines twenty, twenty-three, and twenty-six? Should those lines not be "py27", "py33", and "py34"?